docs/src/docs/asciidoc/user/tiering.adoc

```diff
@@ -73,13 +73,17 @@ include::{sourcedir310}/ehcache-impl/src/test/java/org/ehcache/docs/Tiering.java
 <2> A shortcut to specify 10 entries.
 <3> Only 10 MB allowed. Eviction will occur when full.
 
+NOTE: Byte based heap resources are deprecated and will be removed in a future version.
```

Comment:
We should probably add the note up at line 66 as well, or even just modify the statement at line 65, to be something like "A heap tier can be sized by entries or by size in bytes (though byte-based-sizing of heap tiers is now a deprecated feature)."

Reply:
Yes, line 66 should be updated as well, SO I have updated it to
`A heap tier can be sized by entries or by size(though byte-based-sizing of heap tiers is now a deprecated feature).`